website/docs/tutorial/1-setting-up.md

```diff
@@ -104,7 +104,7 @@ In order to let dbt connect to your warehouse, you'll need generate a keyfile. T
     * **Which API are you using?** BigQuery API
```

Comment:
```suggestion
    * **Which API are you using?** BigQuery API
    * **What data will you be accessing?** Application data (you'll be creating a service account)
```

Also adding this to be more explicit because this tripped me up recently.